frontend/src/common/hooks/useCategoryFilter/common/selectUtils.ts

```diff
@@ -44,7 +44,7 @@ export function buildSelectCategoryView(
   const { categoryFilterId } = config;
 
   // Grab the config for this category.
-  const { pinnedCategoryValues, tooltip } =
+  const { pinnedCategoryValues, pinnedPosition, tooltip } =
```

Comment:
Added configurable `pinnedPosition` to allow pinned values to be positioned at the bottom (e.g. "No publication" is to be listed at the bottom of the Publication menu).